src/Avalonia.Controls/Button.cs

```diff
@@ -41,8 +42,9 @@ public class Button : ContentControl
         /// <summary>
         /// Defines the <see cref="Command"/> property.
         /// </summary>
-        public static readonly StyledProperty<ICommand> CommandProperty =
-            AvaloniaProperty.Register<Button, ICommand>(nameof(Command));
+        public static readonly DirectProperty<Button, ICommand> CommandProperty =
```

Comment:
The subscription to the property changed notification happens in the static constructor. It still works with DirectProperties